Assets/Scripts/Jump.cs

```diff
@@ -29,7 +29,7 @@ void FixedUpdate()
 		if (_isOnGround)
 		{
 			float jumpDirection = Input.GetAxis (jumpAxis);
-			if (Input.GetButtonDown ("Jump") || jumpDirection < 0) {
+			if (jumpDirection > 0) {
```

Comment:
Does `jumpDirection` only exist if the player is jumping?

Reply:
It's a zero unless the button is pressed.

Reply:
Since I have prescribed the button to be a positive action rather than a negative it turns to 1 when the button is pressed